client/src/ClientNotificationManager.ts

```diff
@@ -0,0 +1,33 @@
+import { commands, window } from 'vscode'
+import { type LanguageClient, type Disposable } from 'vscode-languageclient/node'
+
+export class ClientNotificationManager {
+  private readonly _client: LanguageClient
+
+  constructor (client: LanguageClient) {
+    this._client = client
+  }
+
+  buildHandlers (): Disposable[] {
+    const handlers = [
+      this.buildBitBakeNotFoundHandler()
+    ]
+
+    return handlers
+  }
+
+  private buildBitBakeNotFoundHandler (): Disposable {
```

Comment:
Will there also be a "never show again for this workspace" button? Users that don't want full features could be annoyed.

Reply:
LGTM otherwise.